src/main/java/frc/robot/subsystems/superstructure/arm/Arm.java

```diff
@@ -59,7 +59,6 @@ public boolean atSetpoint(final double pivotPositionRots, final double pivotVelo
 
     public enum Goal {
         NONE(0),
-        ZERO(0),
```

Comment:
This is a safe state, it was not related to zeroing the actual mechanism. Not sure why it was removed.

Reply:
It was unused

Reply:
I would keep it as its a safe state.